app/main.py

```diff
@@ -1 +1,19 @@
-# write your code here
+def copy_file(command):
+    operator, file, new_file = command.split()
+    if operator != "cp":
+        return
+    if file == new_file:
+        return
+
```

Comment:
Those two 'if' conditions can be merged in one, by using 'or' operator (operator != "cp" or file == new_file).